eng/pipelines/templates/jobs/perf.yml

```diff
@@ -1,8 +1,8 @@
 parameters:
-- name: ServiceDirectory
+- name: LanguageVersion
   type: string
-  default: ''
-- name: services
+  default: '17'
```

Comment:
Could we add

```
values:
  - 8
  - 11
  - 17
```

To give a convenient selector when running the pipelines while also ensuring a version that is available is used.

Reply:
Sure, let me give this a try and see how it looks.

Reply:
We decided to add the language versions to the displayName instead, since it uses less vertical space than the checkboxes, and we don't use older langauge versions very often.